script.js

```diff
@@ -1,67 +1,159 @@
+  imgDiv.className = "card-image";
+  let img = document.createElement("img");
+  img.setAttribute("src", imgSrc);
+  let h2El = document.createElement("h2");
+  h2El.className = "card-title";
+  h2El.innerText = `${name} - S${twoDigit(season)}E${twoDigit(number)}`;
+  let text = document.createElement("div");
+  text.innerHTML = limitText(summary);
+  imgDiv.appendChild(img);
+  cards.appendChild(imgDiv);
+  imgDiv.after(h2El, text);
+  return cards;
+};
+
+// create show cards
+const createShowCards = ({
+  id,
+  name,
+  image,
+  summary,
+  genres,
+  status,
+  rating,
+  runtime,
+}) => {
+  let mediaQuery = window.matchMedia("(min-width: 650px)");
+  let imgSrc = image
+    ? image.original
+    : "https://www.thermaxglobal.com/wp-content/uploads/2020/05/image-not-found.jpg";
+  let cards = document.createElement("div");
+  cards.classList = "card show-cards";
+  cards.id = id;
+  let imgDiv = document.createElement("div");
+  imgDiv.className = "show-card-image";
+  cards.appendChild(imgDiv);
+  let img = document.createElement("img");
+  img.setAttribute("src", imgSrc);
+  imgDiv.appendChild(img);
+  let generalInfo = document.createElement("div");
+  generalInfo.className = "general-info";
+  let h2El = document.createElement("h2");
+  h2El.className = "card-title";
+  h2El.innerText = name;
+  generalInfo.appendChild(h2El);
+  let text = document.createElement("div");
+  text.className = "card-text";
+  text.innerHTML = mediaQuery.matches ? summary : limitText(summary);
+  let extraInfo = document.createElement("div");
+  extraInfo.className = "extras";
+  let info1 = document.createElement("span");
+  let info2 = document.createElement("span");
+  let info3 = document.createElement("span");
+  let info4 = document.createElement("span");
+  info1.innerText = `Rated: ${rating.average}`;
+  info2.innerText = `Genres: ${genres}`;
+  info3.innerText = `Runtime: ${runtime}`;
+  info4.innerText = `Status: ${status}`;
+  extraInfo.append(info1, info2, info3, info4);
+  imgDiv.after(generalInfo);
+  h2El.after(text, extraInfo);
+  return cards;
 };
 
+// add episode card to the dom
 const makePageForEpisodes = (episodeList) => {
   episodeList.forEach((item) => {
-    cardBody.innerHTML += createEpisodeCards(item);
+    cardBody.appendChild(createEpisodeCards(item));
   });
 };
 
-const search = (e) => {
-  let searchQuery = e.target.value.toLowerCase();
-  let filteredArr = allEpisodes.filter(({ name, summary }) => {
-    return (
-      name.toLowerCase().includes(searchQuery) ||
-      summary.toLowerCase().includes(searchQuery)
-    );
+// add show card to the dom
+const makePageForShows = (episodeList) => {
+  episodeList.forEach((item) => {
+    cardBody.appendChild(createShowCards(item));
   });
-  cardBody.innerHTML = "";
-  makePageForEpisodes(filteredArr);
-  displayLength.innerText = `The number of displayed episodes is ${filteredArr.length}/${allEpisodes.length}`;
 };
 
+// search for shows and episodes
+const search = (e) => {
```

Comment:
When I view your webpage on my browser and I click inside the search bar, it shows me a list with a search history displaying random text that is not related to tv shows or episodes. I have sent a message to you on slack showing a screenshot, to show you what i mean :)